share/completions/mtr.fish

```diff
@@ -0,0 +1,41 @@
+complete -c mtr -x -a "(__fish_print_hostnames)"
+
+complete -c mtr -s h -l help -d 'display this help and exit'
+complete -c mtr -s v -l version -d 'output version information and exit'
+complete -c mtr -s 4 -d 'use IPv4 only'
+complete -c mtr -s 6 -d 'use IPv6 only'
+complete -c mtr -s F -l filename -d 'read hostname(s) from a file' -rF
+complete -c mtr -s r -l report -d 'output using report mode'
+complete -c mtr -s w -l report-wide -d 'output wide report'
+complete -c mtr -s x -l xml -d 'output xml'
+complete -c mtr -s t -l curses -d 'use curses terminal interface'
+complete -c mtr -l displaymode -d 'select initial display mode' -x -a '0 1 2'
```

Comment:
These would really benefit from an explanation.

From the man page I see

> 
Use this option to select the initial display mode: 0 (default)
selects statistics, 1 selects the stripchart without latency
information, and 2 selects the stripchart with latency
information.

So:

```fish
complete -c mtr -l displaymode -d 'select initial display mode' -x -a \
'0\t"Statistics" 1\t"Stripchart without latency" 2\t"Stripchart with latency"'
```

?